### PR TITLE
Add HPE Comware temp limit

### PR DESCRIPTION
**Pull request by evheros**
Changed the Temperature High threshold for a HPE Comware Switch to the values provided by snmp.
This means the actual threshold configured in the switch is used over any predefined/auto calculated thresholds by LibreNMS.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/discovery/sensors/temperature/comware.inc.php [hunk 1]
```diff
@@ -16,10 +16,20 @@
     
 if (!empty($entphydata)) {
     $tempdata = snmpwalk_cache_multi_oid($device, 'hh3cEntityExtTemperature', array(), 'HH3C-ENTITY-EXT-MIB');
-        
+    $hightempdata = snmpwalk_cache_multi_oid($device, 'hh3cEntityExtTemperatureThreshold', array(), 'HH3C-ENTITY-EXT-MIB');
```
[the remark this answers is not in the document]
Comment (laf) [reply includes/discovery/sensors/temperature/comware.inc.php]: I've updated your branch and tested using our test files.

**Comment by murrant** [comment]
@laf I'm not sure this works.  I added the hh3cEntityExtTemperatureThreshold data to the snmprec, but it didn't update the json...  I'll take a look on actual hardware.

**Comment by laf** [comment]
> @laf I'm not sure this works. I added the hh3cEntityExtTemperatureThreshold data to the snmprec, but it didn't update the json... I'll take a look on actual hardware.

I'm pretty sure that was in the snmprec file already.

**Comment by murrant** [comment]
It seems to be working... not sure but I'm not going to dig any deeper :P

**Pull request merged**